user-org-actor/src/main/java/org/sunbird/user/UserReadActor.java

```diff
@@ -48,7 +49,13 @@ public void onReceive(Request request) throws Throwable {
      */
     public void readUserById(Request request) throws BaseException {
         startTrace("readUserById");
-        Response response = userESDao.getUserById((String) request.getRequest().get(JsonKey.USER_ID));
+        Response response = null;
+        try {
+            response = userESDao.getUserById((String) request.getRequest().get(JsonKey.USER_ID));
+        } catch (Exception e) {
```

Comment:
Add logger in catch block to print sacktrace, to know what went wrong.

Reply:
done